spec/models/payroll/payment_csv_row_spec.rb

```diff
@@ -1,13 +1,13 @@
 require "rails_helper"
 
-RSpec.describe Payroll::ClaimCsvRow do
-  subject { described_class.new(claim) }
+RSpec.describe Payroll::PaymentCsvRow do
+  subject { described_class.new(payment) }
   let(:claim) { build(:claim) }
 
   describe "to_s with an approved claim that has a payment" do
     let(:row) { CSV.parse(subject.to_s).first }
     let(:payment_award_amount) { BigDecimal("1234.56") }
-    let!(:payment) { build(:payment, award_amount: payment_award_amount, claim: claim) }
```

Comment:
👍 this bothered me ever so slightly in the last PR, glad to see this refactoring means it can go.